docs: Add Managing Goose Permissions Guide (#1421)
block/goose: documentation/docs/guides/goose-permissions.md

---
sidebar_position: 3
title: Goose Permissions
---
import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

# Managing Goose Permissions

Goose’s **permissions** determine how much autonomy it has when modifying files, using extensions, and performing automated actions. By selecting a permission mode, you have full control over how Goose interacts with your development environment.

## Permission Modes

| Mode        | Description | Best For |
|------------|------------|---------|
| **Auto Mode** | Goose can modify files, use extensions, and delete files **without requiring approval**. | Users who want **full automation** and seamless integration into their workflow. |
| **Approve Mode** | Goose **asks for confirmation** before modifying, creating, deleting files and before using extensions. | Users who want to **review and approve** changes and extension use before they happen. |
| **Chat Mode** | Goose **only engages in chat**, with no extension use or file modifications. | Users who prefer a **conversational AI experience** without automation. |

:::warning
    `Auto Mode` is applied by default unless you specify otherwise.
:::

## Configuring Goose Permissions

> To configure your chosen goose permissions, run `goose configure` in the CLI or visit the `Provider Settings` page in the Goose Desktop.

Reviewer: ```suggestion
Here's how to configure your chosen goose permissions:
```
